posts/project-mycorrhiza/index.md

```diff
@@ -0,0 +1,191 @@
+beneficial to avoid having to re-parse things.
+[Log Detective](https://log-detective.com), a project to help parse logs and provide information on why builds failed, can
+Log Detective, a project to help parse logs and provide information on why builds failed, can
+also be integrated to help their project get more logs to train with (opt-in of course).
+
+#### Product overview
+
+If you manage a product and for instance use Testing Farm you know the pain of waiting for the
+results to happen. Especially so when you are waiting on test runs within several PRs/MRs.
+
+Getting a better view for the project view to display all the pipelines for the repository
+would not only help get a holistic view for how things are going, but allow you to instantly
+see and even be notified of the results.
+
+#### Filters, search, & sorting
+
+In the current Packit dashboard you can even see a what's supposed to be search field that
+allows you to search for the forge (like GitHub), git organizations, and git repositories.
+If you've tried to use this you've quickly realized this needs an exact name and is more of a
+"quick-link" to the respective view than a search.
+
+![Search-like inputs in the dashboard's project view](img/project-search.png)
+
+What is seen in most other tables throughout as well, including the project view, is that we
+don't have any filters available. The sorting is also done by most recent first, but what if
+you want to sort by PR for example? That is missing in the current implementation.
+
+Having a search to get the results quickly would also be good, which would be similar to filter
+but just a free-form field basically.
+
+## Research and interviews
+
+So what's next? There is a lot of great information out there already, such as interviews
+by the [Red Hat UXD team](https://www.uxd-hub.com/). We will be going through
+data reported for the general lacking features developers want or are frustrated with,
+as well as following-up with Packit users with a more Packit-scoped interview to
+see how on the goal our research is. Depending on this research, which we hope to be done
+beginning of 2025 in a worst-case scenario, we will start to plan out the work
+to make the Packit service and Packit dashboard be the best it could be.
+
+We will of course continue to improve everything until then, Packit dashboard will get gradual
+improvements throughout the rest of the year and Packit will continue to work on the goals they've
+announced for the quarter.
+
+## TL;DR
+
+Project Mycorrhiza is a temporary name to encompass the work relating to what the next generation
+Packit dashboard could look like by seeing how developer time can be saved, minimizing
+complexity, reducing context switching, and providing results faster. Research going into this
+will likely improve Packit service as well. 'Mycorrhiza' means a symbiotic network between plants
+and fungi to share nutrients, fats, and sugars between each other to grow.
```

Comment:
In the beginning, we've also discussed that this might not be strictly tight to Packit (this being a dashboard for other tools/services and/or results not triggered by Packit and/or not built on Packit's dashboard) -- is this something we want to cover here or we ended with Packit only use case?